library/ziti.c

```diff
@@ -622,21 +613,21 @@ static const ziti_version sdk_version = {
         .build_date = to_str(BUILD_DATE)
 };
 
-const ziti_version *NF_get_version() {
+const ziti_version *ziti_get_version() {
     return &sdk_version;
 }
 
 static void grim_reaper(uv_prepare_t *p) {
-    nf_context nf = p->data;
+    ziti_context nf = p->data;
```

Comment:
another variable that's used in so many places... maybe you left it on purpose? :)

Reply:
not chasing individual variables